src/pages/actionitems/actionitems.ts

```diff
@@ -140,7 +140,7 @@ console.log(element);
                         nq.threadName,
                         nq.subThreadName,
                         nq.questionText,
-                        nq.answers[nq.answers.length - 1].answer,
+                        // nq.answers[nq.answers.length - 1].answer,
```

Comment:
it's ok to delete lines we're not using rather than commenting them out, don't think we'll be going back to using the answer field any more.